src/redux/books/bookSlice.js

```diff
@@ -12,7 +31,7 @@ export const booksSlice = createSlice({
       state.books.push(action.payload);
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the state is immutable, instead of using `Array.push()` you can replace your code with `state.books = [...state.books, action.payload]`

Reply:
@Ismailco thank you! I've implemented your suggestion. I appreciate it !